docs/admin/ServerSideKeygen.adoc

```diff
@@ -0,0 +1,157 @@
+= ServerSide Key Generation for EE
+Author <cfu@redhat.com>
+:toc:
+:imagesdir: images
+
+== Overview
+
+Many newer versions of browsers, including Firefox v69 and up, as well as Chrome, have removed the functionality to generate PKI keys and the support for CRMF for key archival. While CLIs such as CRMFPopClient (see "CRMFPopClient --help") or pki (see "pki client-cert-request --help") could be used as a workaround on the Fedora or RHEL platforms, clients on non-RedHat platforms have been largely left out. 
+
+Server-Side Keygen enrollment has been around for a long time since the introduction of Token Key Management System (TMS), where keys could be generated on KRA instead of locally on smart cards. In this release, we adopt a similar mechanism to resolve the browser keygen deficiency issue.   Keys are generated on the server (KRA to be specific) and then transferred securely back to the client in PKCS#12.
```

Comment:
Do we use "PKI" instead of "Dogtag" now?

Reply:
also, I just checked the bug.  The "Fixed in Verson" field says "pki-core-10.5.18-6.el7", so should I put down 10.5 instead?  Or, is there a better way to find the exact release of a feature?

Reply:
We're using PKI as the generic product name in the common docs since Dogtag is strictly upstream and RHCS is strictly downstream. We can probably say "In PKI 10.5, 10.9, or later versions, ..." I don't think we need to mention the patch number (the third digit) or the release number/platform since people are expected to upgrade to the latest release in PKI x.y version.